# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../glibc/2.3.4-kd.h.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

No idea when this was introduced and for what - but it prevents the definition
of vital basic types needed thruout many kernel headers which breaks some
applications depending of the order the headers are included (e.g. dosemu).

  - Rene Rebe <reneÂ@exactcode.de>

--- libc/sysdeps/unix/sysv/linux/sys/kd.h.orig	2000-01-01 00:00:00.000000000 +0100
+++ libc/sysdeps/unix/sysv/linux/sys/kd.h	2004-12-12 11:10:46.905723872 +0100
@@ -20,9 +20,11 @@
 #define _SYS_KD_H	1
 
 /* Make sure the <linux/types.h> header is not loaded.  */
+/* but not on our systems -ReneR
 #ifndef _LINUX_TYPES_H
 # define _LINUX_TYPES_H	1
 #endif
+*/
 
 #include <linux/kd.h>
 
